server/segment_rpc.go

```diff
@@ -857,14 +857,15 @@ func genPayment(ctx context.Context, sess *BroadcastSession, numTickets int) (st
 		protoPayment.TicketSenderParams = senderParams
 
 		ratPrice, _ := common.RatPriceInfo(protoPayment.ExpectedPrice)
-		clog.Infof(ctx, "Created new payment - manifestID=%v sessionID=%v recipient=%v faceValue=%v winProb=%v price=%v numTickets=%v",
+		clog.Infof(ctx, "Created new payment - manifestID=%v sessionID=%v recipient=%v faceValue=%v winProb=%v price=%v numTickets=%v balance=%v",
```

Comment:
Soft suggestion - using the `Info` helper rather than `Infof` will lead to a neater message string and better parameter locality, eg

```golang
clog.Info(ctx, "Created new payment",
  // manifestID should already be set as part of the ctx elsewhere
  "session", sess.OrchestratorInfo.AuthToken.SessionId,
  "recipient", batch.Recipient.Hex(),
  "faceValue",  eth.FormatUnits(batch.FaceValue, "ETH"),
  "winProb", batch.WinProbRat().FloatString(10),
  "numTickets", numTickets,
  "balance", sess.Balances.Balance(batch.Recipient, core.ManifestID(sess.PMSessionID)).FloatString(3)
)
```

Reply:
Good idea. Updated.